src/index.spec.js

```diff
@@ -28,6 +28,14 @@ const data = deepFreeze({
       username: 'Administrator',
       password: 'pas:sw@ord'
     }
+  },
+  NFS: {
+    string: 'nfs://192.168.100.225:media/nfs',
```

Comment:
Ah yes there is practice, but it is not under responsibility of the parser, the test data could just be SOME_HOST:SOME_PATH in fact